cardinal/cardinal.go

```diff
@@ -175,7 +145,7 @@ func CreateMany(wCtx engine.Context, num int, components ...types.Component) (en
 	for _, comp := range components {
 		c, err := wCtx.GetComponentByName(comp.Name())
 		if err != nil {
-			return nil, ErrComponentNotRegistered
+			return nil, eris.Wrap(err, "failed to create entity because component is not registered")
```

Comment:
This error and the identical one below should be component.ErrComponentNotRegistered

Reply:
The error returned by `wCtx.GetComponentByName(...)` is `ErrComponentNotRegistered` this just use the wrap feature to add additional context that this fails during the creation of entity.

Reply:
Ah, my mistake.